public/static/locales/fr/common.json

```diff
@@ -12,7 +12,7 @@
       "link": "/fr/form-builder/support",
       "desc": "Soutien"
     },
-    "ariaLabel": "[FR] Footer"
+    "ariaLabel": "Footer"
```

Comment:
Missing French translation here. Not sure how it would be translated in French but probably something like "Bas de page"

Reply:
Updated using your translation --- it's for an Aria label so will be fine.